config/storybook-addon-knobs.d.ts

```diff
@@ -0,0 +1,29 @@
+declare module 'storybook-addon-knobs' {
```

Comment:
I'm not actually sure how declaration files work when included with modules.

Using redux definitions as reference:
https://github.com/reactjs/redux/blob/master/index.d.ts

Seems like we don't need the `declare module` wrapper?